test/README.md

```diff
@@ -75,8 +75,8 @@ The recommended way to do this yourself is to use python's `virtualenv`
 You can set up virtualenv for all these environments:
 
 ```console
-virtualenv -p python3.8 venv3.8
-virtualenv -p python3.9 venv3.9
+virtualenv -p python3.12 venv3.8
+virtualenv -p python3.12 venv3.9
 ```
 
 To activate one of these (the example uses 3.8), run the tests, and deactivate:
```

Comment:
I see more mentions of 3.8 here and in other places that should probably be removed.